src/commands/buildCommands/configs/serverAppConfig.js

```diff
@@ -0,0 +1,114 @@
+// Require modules
```

Comment:
Copied from [`mwp-config`](https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-config/webpack/serverAppConfig.js). Edited for Webpack 4